plugin/pkg/scheduler/factory/factory.go

```diff
@@ -458,24 +449,24 @@ func (c *configFactory) onPvcDelete(obj interface{}) {
 }
 
 func (c *configFactory) invalidatePredicatesForPvc(pvc *v1.PersistentVolumeClaim) {
-	if pvc.Spec.VolumeName != "" {
-		c.equivalencePodCache.InvalidateCachedPredicateItemOfAllNodes(maxPDVolumeCountPredicateSet)
-	}
+	// TODO(harry): make all predicate key to be global const, e.g. predicates.NoVolumeZoneConflict
```

Comment:
Can you add a comment that we need to do this here because the ecache uses PVC uid to hash the pod

Reply:
I guess for that same reason, VolumeBind predicate also needs to be invalidated

Reply:
ack, just updated.